mos-platform/geos-cbm/geos.h

```diff
@@ -0,0 +1,20 @@
+/***********************************************************************
+ * geos.h
+ *
+ * This file contains definitions for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#ifndef GEOS_H
+#define GEOS_H
+
+#include "geos_constants.h"
```

Comment:
This should begin with the usual C++ wrappers to make sure that the symbols defined within use the C API in C++.
See most of the other C headers in the SDK for how to do this.